data/constants.py

```diff
@@ -29,6 +29,5 @@
 UBUNTU_18_04           = "From ubuntu:18.04"
 UBUNTU_20_04           = "From ubuntu:20.04"
 LOGS                   = os.path.join(FRAMEWORK_PATH, "logs")
-GRAMINE_BRANCH         = "master"
 GRAMINE_VERSION        = "v1.3.1"
```

Comment:
Can this tag stay the same when we plan to test with master? Should we take this also as parameter for future use cases where we might want to run with a different tag?

Reply:
We can give the commit as master from jenkins and it will work in similar way